ui/app/routes/vault/cluster/access/mfa/methods/method/edit.js

```diff
@@ -0,0 +1,3 @@
+import Route from '@ember/routing/route';
+
+export default class MfaMethodEditRoute extends Route {}
```

Comment:
Doesn't this route need the model of the MFA being edited?

Reply:
The parent method route loads the model and it gets passed down to the child. It's funny though I can't seem to find anywhere in the documentation that mentions that if you do not define a model hook in a child route it receives the parent model. Maybe that's not a thing but it's definitely getting the model from the parent.

Reply:
And here I thought it was a standard practice in ember that model gets passed down to child. 😆